transports/jms/src/main/java/org/mule/transport/jms/transformers/JMSMessageToObject.java

```diff
@@ -58,7 +58,7 @@ public Object transformMessage(MuleMessage message, String outputEncoding) throw
         {
             if (logger.isDebugEnabled())
             {
-                logger.debug("Source object is " + ClassUtils.getSimpleName(message.getPayload().getClass()));
+                logger.debug("Source object is " + getSimpleName(message.getPayload().getClass()));
```

Comment:
Don't you have the same NPE here?

Reply:
I have tested it and NPE doesn't occur here since a null payload is always represented by a NullPayload instance.